### PR TITLE
Added lineedits for certificates

### LINKED ISSUE #77: Support AWS Keyspaces
Greetings.

As I can see, initially the plugin worked with Cassandra only. And the ability to work with AWS Keyspaces was added in [this](https://github.com/HadesArchitect/GrafanaCassandraDatasource/pull/65) PR.

Once appropriate PR was merged, I've built the code (with the necessary [Amazon certificate](https://www.amazontrust.com/repository/AmazonRootCA1.pem) included) by myself (as no official release with necessary commits created yet). 

The built plugin installed successfully. 

With help of `cqlsh` tool, I can successfully connect to AWS Keyspaces.
So I wouldn't expect any problems to do the same with help of the plugin if the same is in use:
- credentials;
- Amazon certificate;
- AWS Keyspace endpoint;
- server, from which you're trying to reach AWS Keyspaces.

Yet, a Datasource, that uses the plugin, can't connect to AWS Keyspaces.
No logs in Grafana.
And I can't find any ability to add DEBUG level to see, what's exactly wrong with my connection.

Does anyone already have a positive experience connecting this plugin to AWS Keyspaces?

Thank you.

### PR DESCRIPTION
**Pull request by futuarmo**
If you will add path to certificates (cert and root of ca), plugin will use it to connect to database.
If you will set only cert or only root - it will be ignored
Fixes #77

### CODE REVIEW

**Comment by futuarmo** [comment]
@HadesArchitect please, check it with Astra and AWS

File: backend/read_tls.go [hunk 1]
```diff
@@ -9,7 +9,7 @@ import (
 )
 
 func PrepareTLSCfg(certPath string, rootPath string, caPath string) (*tls.Config, error) {
-	tlsConfig := &tls.Config{InsecureSkipVerify: true}
+	tlsConfig := &tls.Config{InsecureSkipVerify: false}
```
Comment (HadesArchitect) [inline backend/read_tls.go:12]: With the self-signed certificates, that must be configurable

Comment (futuarmo) [reply backend/read_tls.go]: Yes, it's reasonable. I turned it on and created task for implementing checkbox to configure this

**Comment by futuarmo** [comment]
Looks like we have some problems with github actions

**Comment by HadesArchitect** [comment]
Sure I'll check it

**Comment by HadesArchitect** [comment]
Some kind of golang issues, looks like https://github.com/HadesArchitect/GrafanaCassandraDatasource/pull/84/checks?check_run_id=1963671321#step:6:4

**Comment by HadesArchitect** [comment]
Looks related https://github.com/golang/go/issues/31997#issuecomment-782864390

**Comment by futuarmo** [comment]
> Looks related https://github.com/golang/go/issues/31997#issuecomment-782864390
> 

I'll look it, thanks

**Comment by HadesArchitect** [comment]
https://blog.golang.org/go116-module-changes I think that's the issue. Do you need any help with that?

**Comment by futuarmo** [comment]
> 
> 
> https://blog.golang.org/go116-module-changes I think that's the issue. Do you need any help with that?

I'll try to fix it today

**Pull request merged**